engine/src/main/java/org/terasology/rendering/nui/layers/ingame/metrics/DebugOverlay.java

```diff
@@ -183,10 +180,9 @@ public boolean isEscapeToCloseAllowed() {
     }
 
```

Comment:
/**
Move forward through the MetricsMode instances and displays the content of the next available one.
*/